Dockerfile

```diff
@@ -23,9 +23,7 @@ COPY --from=builder /go/src/sigs.k8s.io/ibm-powervs-block-csi-driver/bin/ibm-pow
 ENTRYPOINT ["/ibm-powervs-block-csi-driver"]
 
 
-FROM registry.access.redhat.com/ubi8/ubi:8.5 AS rhel-base
-RUN yum --disableplugin=subscription-manager -y install httpd php \
-  && yum --disableplugin=subscription-manager clean all
-RUN clean-install ca-certificates e2fsprogs mount udev util-linux xfsprogs
-COPY --from=builder /go/src/sigs.k8s.io/ibm-powervs-block-csi-driver /ibm-powervs-block-csi-driver
+FROM quay.io/centos/centos:stream9 AS centos-base
```

Comment:
stream9 is still in beta stage, lets stick to stream8 for now.

Reply:
There were some security vulnerabilities in stream8.
that's why I moved to stream9 and the vulnerabilities are not seen now.
Do you think we still need to move to stream8.

Reply:
stream changed to stream8 as discussed